vispy/geometry/curves.py

```diff
@@ -0,0 +1,343 @@
+# ----------------------------------------------------------------------------
+#  Anti-Grain Geometry (AGG) - Version 2.5
+#  A high quality rendering engine for C++
+#  Copyright (C) 2002-2006 Maxim Shemanarev
+#  Contact: mcseem@antigrain.com
+#           mcseemagg@yahoo.com
+#           http://antigrain.com
+#
+#  AGG is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
```

Comment:
We cannot use GPL-licensed works, it is incompatible with our BSD license. @rougier did you get permission to relicense as BSD from the AGG folks?